### DISCUSSION TITLE
not working on iphone 6s ios 15.0.1

### DISCUSSION THREAD

**Discussion by legofan123456** [utmapp/UTM#3162] [UTM Help (iOS)]
i get a error opening the app

"your version of ios does not support running VMs while unmodified. you must either run utm jailbreak or with a remote debugger attached"
would like some help
thanks

**Comment by Andrupka** [comment]
did you read the instruction? get a second iphone and run jitterbug or jailbreak. you can also do xcode tethering but i havent had any success. always read instructions before reporting.
